tests/test_runner/test_runner.py

```diff
@@ -1487,7 +1511,7 @@ def before_val_iter(self, runner, batch_idx, data_batch=None):
         val_interval_results = []
         val_interval_targets = [5] * 10 + [2] * 2
 
-        @HOOKS.register_module()
+        @HOOKS.register_module(force=True)
```

Comment:
Why do we need to set `force=True`?

Reply:
Some toy classes with the same name are defined in different unit tests. If `force=False`, the class defined in current unit test will not be registered. If we refactor the unit test and make sure each unit test will clean the registered class, there is no need to set `force=True` anymore.